install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -156,30 +164,32 @@ it in OpenShift:
 
 [source,yaml]
 ----
-apiVersion: "v1"
-kind: "PersistentVolume"
+apiVersion: v1
+kind: PersistentVolume
 metadata:
-  name: "gluster-default-volume" <1>
+  name: gluster-default-volume <1>
 spec:
   capacity:
-    storage: "2Gi" <2>
-  accessModes:
-    - "ReadWriteMany"
-  glusterfs: <3>
-    endpoints: "glusterfs-cluster" <4>
-    path: "myVol1" <5>
+    storage: 2Gi <2>
+  accessModes: <3>
+    - ReadWriteMany
+  glusterfs: <4>
+    endpoints: glusterfs-cluster <5>
+    path: myVol1 <6>
     readOnly: false
-  persistentVolumeReclaimPolicy: "Recycle"
+  persistentVolumeReclaimPolicy: Recycle
 ----
 <1> The name of the volume. This will be how it is identified via
 link:../../architecture/additional_concepts/storage.html[persistent volume
 claims] or from pods.
 <2> The amount of storage allocated to this volume.
-<3> This defines the volume type being used, in this case the *glusterfs*
+<3> `accessModes` are used as labels to match a PV and a PVC. They currently
+do not define any form of access control.
+<4> This defines the volume type being used, in this case the *glusterfs*
 plug-in.
-<4> A reference to the endpoints object that defines the Gluster cluster,
+<5> A reference to the endpoints object that defines the Gluster cluster,
 created in link:#creating-gluster-endpoints[Creating Gluster Endpoints].
-<5> This is the Gluster volume that will be used, as defined on your Gluster
+<6> This is the Gluster volume that will be used, as defined on your Gluster
```

Comment:
Don't mix voices like 'This is' in one bullet and 'Defines bla bla' in another, pick one and be consistent through out.  I prefer the voice like 'Defines bla bla'